### PR TITLE
docs: update links and spelling in README.md

### PR DESCRIPTION
**Pull request by vladdoster**
- add missing links to official websites
- fix markdown formatting for consistency with rest of document
- correct spelling of package targets to match previous reference in document (e.g., consistency)
- make links formatting consistent
- remove extra new line at end of document

**Note**: `diagram.svg` is missing

Closes #(issue)

### CODE REVIEW

**Comment by xPolar** [comment]
The team are no longer taking pull requests for typo fixes or such, this would fit better under the [Found a typo?](https://github.com/spacedriveapp/spacedrive/discussions/142) discussion. In the future I would say don't make any pull request or issues for things such as adding links to the images or making typo fixes, just reply to the discussion above or create a new one.

This will likely be closed soon.

File: README.md [hunk 1]
```diff
@@ -118,25 +118,25 @@ This project is using what I'm calling the **"PRRTT"** stack (Prisma, Rust, Reac
...
-- `android`: A [Kotlin](#) Native binary (planned).
+- `macOS`: A [Swift](https://developer.apple.com/swift) Native binary for MacOS system extensions.
+- `iOS`: A [Swift](https://developer.apple.com/swift) Native binary (planned).
+- `Windows`: A [C#](https://docs.microsoft.com/en-us/dotnet/csharp) Native binary (planned).
+- `Android`: A [Kotlin](https://kotlinlang.org) Native binary (planned).
 
 ### Repo structure:
-![Visualization of the codebase](./diagram.svg)
 
+![Visualization of the codebase](./diagram.svg)
```
Comment (vladdoster) [inline README.md:142]:
```suggestion
![Visualization of the codebase](./diagram.svg)
```

Was unable to find this in the repository from a cursory search. However, did see the diagram workflow is disabled but didn't seem to have any related logic -- but possibly related/cruft?

Comment (xPolar) [reply README.md:142]: Look at the comment I just made, use our discussions.

**Comment by vladdoster** [comment]
@xPolar I suggest this is made clear in the CONTRIBUTING.md. 

https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md#solve-an-issue

**Comment by jamiepine** [comment]
id be happy to accept this PR, sorry @xPolar I know I told you to deny spelling PRs, but this one was actually helpful with all the link fixes and being well documented.

can we merge this? i can't work it out on mobile😩

**Pull request closed without merging**